web/Makefile

```diff
@@ -11,12 +11,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: blob
+include ../Makefile.INCLUDE
 
-blob:
-	$(MAKE) -C blob
+all: blob/files.go
+
+SUFFIXES:
+
+# Do not embed map files and the non-minified bootstrap files.
```

Comment:
I think this comment is meant just as a note that the embed-static.sh script won't include the files. @beorn7 might be able to explain this in more detail. This is the commit which introduced the line: https://github.com/prometheus/prometheus/commit/7e5745f